libs/device/pxtsnippets.json

```diff
@@ -1,118 +1,106 @@
+[
+    {
+        "name": "Create a Sprite",
+        "namespace": "sprites",
+        "group": "Create",
+        "label": "Create a sprite...",
+        "outputType": "blocks",
+        "initialOutput": "let $spriteName = sprites.create($spriteImage, $spriteKind)",
+        "questions": [
+            {
+                "title": "Draw your sprite",
+                "inputs": [
+                    {
+                        "answerToken": "spriteImage",
+                        "type": "spriteEditor",
+                        "defaultAnswer": "img`\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n. . . . . . . . . . . . . . . .\n`"
+                    }
+                ],
+                "hint": "Sprites are 2D characters and objects used in all MakeCode Arcade games. Click and drag to draw your own sprite.",
+                "goto": {
+                    "question": 1
                 }
-            ],
-            "goto": {
-                "question": 1
-            }
-        },
-        {
-            "title": "What kind of sprite should this be?",
-            "inputs": [
-                {
-                    "answerToken": "spriteKind",
-                    "defaultAnswer": "SpriteKind.Player",
-                    "type": "dropdown",
-                    "options": {
-                        "SpriteKind.Player": "Player",
-                         "SpriteKind.Enemy": "Enemy",
-                        "SpriteKind.Food": "Food",
-                        "SpriteKind.Projectile": "Projectile"
-                    } 
-                }
-            ],
-            "goto": {
-                "question": 4,
-                "parameters": {
-                    "spriteKind": 0
-                }
-            }
-        },
-        {
-            "title": "How many lives should your player have?",
-            "inputs": [
-                {
-                    "answerToken": "gameLives",
-                    "defaultAnswer": 3,
-                    "max": 20,
-                    "type": "number"
+            },
+            {
+                "title": "Name your sprite",
+                "inputs": [
+                    {
+                        "answerToken": "spriteName",
+                        "defaultAnswer": "mySprite",
+                        "type": "text"
+                    }
+                ],
+                "output": "",
+                "hint": "Naming sprites is important because then you can refer to your sprite by name elsewhere in your program.",
```

Comment:
```suggestion
                "hint": "Naming sprites is important because you can use the name to refer to your sprite elsewhere in your program.",
```